features/prompt-registry/function-calling.mdx

```diff
@@ -4,28 +4,31 @@ icon: "function"
 ---
 
 
-The Prompt Registry supports the [OpenAI functions](https://platform.openai.com/docs/guides/function-calling) format. You can create a prompt using functions both visually and programmatically.
+The Prompt Registry supports the [OpenAI tools](https://platform.openai.com/docs/guides/function-calling) format. You can create a prompt using tools both visually and programmatically.
 
 Be sure to read [this OpenAI guide](https://platform.openai.com/docs/guides/function-calling) for more context.
 
 ### Creating Visually
 
-Functions can be defined, called, and set up visually through the Prompt Registry.
+Tools can be defined, called, and set up visually through the Prompt Registry.
 
-<img src="/images/create-function.gif" />
+<video controls="controls">
```

Comment:
This video doesn't actually show the process of creating a function. 

Mouse is moving around a lot, doesn't look professional. Also spend more time on the interactive builder, not the JSON

Also please don't use the promptlayer sql template which contains our internal DB schema